pkgs/tools/misc/yt-dlp/default.nix

```
@@ -38,16 +34,13 @@ buildPythonPackage rec {
         ++ lib.optional phantomjsSupport phantomjs2;
     in [ ''--prefix PATH : "${lib.makeBinPath packagesToBinPath}"'' ];
 
```

```suggestion

  # build_lazy_extractors assumes this directory exists
  # but it is not present in the pypi package
  postPatch = ''
    mkdir -p ytdlp_plugins/extractor
  '';
```

this should fix lazy extractors, taken from #133762